src/js/tabs/configuration.js

```diff
@@ -311,6 +311,27 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
         // translate to user-selected language
         i18n.localizePage();
 
+        var gyro_align_content_e = $('.tab-configuration .gyro_align_content');
+        var legacy_gyro_alignment_e = $('.tab-configuration .legacy_gyro_alignment');
+        var legacy_accel_alignment_e = $('.tab-configuration .legacy_accel_alignment');
+
+        // Hide the new multi gyro element by default
+        gyro_align_content_e.hide();
+
+        // If we have the newer MSP API, and USE_MULTI_GYRO is defined in the target, show the new element, while hiding the old ones.
+        if (semver.gte(CONFIG.apiVersion, "1.41.0") &&
+            SENSOR_ALIGNMENT.use_multi_gyro == 1) {
+            gyro_align_content_e.show();
+            legacy_gyro_alignment_e.hide();
+            legacy_accel_alignment_e.hide();
+        }
+
+        // As the gyro_to_use does not have a 'DEFAULT' 0th element enum, the 0th element 'First' is excluded from here.
+        var gyros = [
+            'Second',
+            'Both'
+        ];
```

Comment:
This text must go in the `messages.json` file to translate them.

Reply:
@McGiverGim hmm I am using these in the same manner as the alignment, escprotocols etc - it doesn't seem they are hooked into i18n system. can you give an example how I would do so?

Reply:
Yes, the major part has been translated, but some of them remain in the code. We usually translate them when adding to the select, but you can do in the same way when creating the array: https://github.com/betaflight/betaflight-configurator/blob/974e622b73aa744fe5e4d5545d1dcb65e28d1f3f/src/js/tabs/adjustments.js#L292-L294

Reply:
Of course, if we translate the second and both we need to translate the first too ;)